javascripts/controllers/FavoritesCtrl.js

```diff
@@ -3,21 +3,41 @@
 app.controller("FavoritesCtrl", function($location, $rootScope, $scope, ContactService){
 	$scope.contacts = [];
 
-	// const getContacts = () => {
-	// 	ContactService.getFavoriteContacts($rootScope.uid).then((results) =>{
-	// 		$scope.contacts = results;
-	// 	}).catch((err) =>{
-	// 		console.log("error in getFavoriteContacts", err);
-	// 	});
-	// };
+	const getContacts = () => {
+		ContactService.getFavoriteContacts($rootScope.uid).then((results) =>{
+			$scope.contacts = results;
+		}).catch((err) =>{
+			console.log("error in getFavoriteContacts", err);
+		});
+	};
 
-	// getContacts();
+	$scope.deleteContact = ( contactId ) => {
+		ContactService.deleteContactInFb(contactId).then((results) => {
+			getContacts();
+		}).catch((err) => {
+			console.log('error in deleteContactInFb:', err);
+		});  
+	};
 
-	// $scope.deleteContact = (contactId) => {
-	// 	ContactService.deletecontact(contactId).then((result) => {
-	// 		getContacts();
-	// 	}).catch((err) =>{
-	// 		console.log("error in deleteContact", err);
-	// 	});
-	// };
+	$scope.changeFav = (contact, contactId) => {
+		contact.favorite = contact.favorite ? false: true;
```

Comment:
the property here that you specify is 'favorite'. this is different from the property that you use to make the uniform contact information in your createContactObject function (in there, the property name is 'isFavorite')